ncl/ncl_entries/general_applied_math.ipynb

```diff
@@ -0,0 +1,713 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    abs_output = abs(input_value)
</pre><p> </p><p>Is there a reason not to use the numpy version (np.abs or np.absolute)? Seems more consistent with other stuff, but I might be missing something.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
It is a better way to translate the array so I'll change that, but I think not requiring the external package when it is isn't needed for simple math can make it complicated. This notebook uses the standard library when it can for single inputs since it doesn't need any additional work or package wrangling.

Reply:
I think we'll probably want to include the numpy versions of functions where available given the audience (lots of arrays and place where you'd want to use a numpy ufunc if possible), but including both probably doesn't hurt.

This is what I was getting at below in a few places as well.

Reply:
We can log an issue and separate out this conversation, but I feel pretty strongly that we should be at least including examples of functions that operate on arrays.

Reply:
I expanded the functions to include working with lists and operating with `numpy` functions that return arrays

Reply:
It looks like they're still missing from the NCL Index and the receipt as well.  I think it's worth making the change on the index at least.  The receipt probably isn't a bad idea though and would be more consistent w/ the trig functions.

Reply:
The NCL index now references `abs() or numpy.abs()`, but I have expanded the receipt to include math and numpy within the v2 https://github.com/NCAR/geocat-applications/issues/79, so I can resolve that with the next PR